src/Language/Docker/Parser.hs

```diff
@@ -107,7 +107,7 @@ pairs = pair `sepBy1` spaces1
 label :: Parser Instruction
 label = do
     reserved "LABEL"
-    p <- pairs
+    p <- envPairs
```

Comment:
Using `envPairs` for `LABEL` now looks a bit misleading. Would you consider renaming of `envPairs`?

Reply:
Sure, that makes sense